sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -340,6 +340,11 @@ static void _send_to_iface(kernel_pid_t iface, gnrc_pktsnip_t *pkt)
         gnrc_pktbuf_release(pkt);
         return;
     }
+#ifdef MODULE_NETSTATS_IPV6
+    if_entry->stats.tx_success++;
+    if_entry->stats.tx_bytes += gnrc_pkt_len(pkt->next);
```

Comment:
not sure if it's available here, but why not use the length field of the `ipv6` header? Isn't that more accurate than `gnrc_pkt_len(pkt->next)`?